server/src/main/kotlin/com/github/neho4y/comment/service/impl/CommentServiceImpl.kt

```diff
@@ -0,0 +1,49 @@
+package com.github.neho4y.comment.service.impl
+
+import com.github.neho4y.comment.domain.Comment
+import com.github.neho4y.comment.domain.repository.CommentRepository
+import com.github.neho4y.comment.model.CommentCreationDto
+import com.github.neho4y.comment.service.CommentService
+import com.github.neho4y.feedback.domain.repository.FeedbackRepository
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+
+@Service
+class CommentServiceImpl(
+    private val commentRepository: CommentRepository,
+    private val feedbackRepository: FeedbackRepository
+) : CommentService {
+    override fun addComment(commentCreationDto: CommentCreationDto) {
+        val feedback = feedbackRepository.findById(commentCreationDto.feedbackId).orElseThrow()
+        val comment = Comment(
+            authorId = commentCreationDto.authorId,
+            feedbackId = feedback,
+            isDeleted = false,
+            messageDate = LocalDateTime.now(),
+            isUnread = true,
+            messageText = commentCreationDto.messageText,
+            messageType = commentCreationDto.messageType
+        )
+        commentRepository.save(comment)
+    }
+
+    override fun getComments(feedbackId: Long): List<Comment> {
+        return commentRepository.findByFeedbackIdAndIsDeletedFalse(
```

Comment:
Consider receiving feedback from parameters, which will be validated on the call site. Because it's not quite the responsibility of CommentService to check feedback existence. But it is contraversial question and your implementation is all ok.